hub/powertoys/run.md

```diff
@@ -240,6 +240,21 @@ Examples:
 - `) calendar week::3/27/2022` to show the calendar week for a date value.
 - `) unix epoch::3/27/2022 10:30:45 AM` to convert the given time and date value into a Unix epoch timestamp.
 
+#### Custom formats
+The plugin has a setting were you can define custom formats. The setting is a multiline text box were you have ta add one format per line.
+The syntax is `<format name>=<format pattern>` for using the local time and `<format name>=UTC:<format pattern>` for using the Universal Time Convention (UTC).
```

Comment:
Are these blocks intended to be separate paragraphs? If so, you should add an additional line break after each. Otherwise, they'll be combined into a single large paragraph.

Reply:
I will add a line break.